CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 #Change Log
 All notable changes to this project starting with the 0.6.0 release will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org)
 
+## 0.13.6
+- NOTICE:
```

Comment:
I think that either a sentence needs to be added to the effect that the update message has been changed to this, or an additional point saying as much. Also needs a `### Changed` header like the other versions have.